### DISCUSSION TITLE
I am unable to set a custom db(mongodb) name for flask - limiter :( please help

### DISCUSSION THREAD

**Discussion by Vishwamithra37** [alisaifee/flask-limiter#420] [Q&A]
I have 3 different flask applications with their own rate limiters and possibly overlapping route names, I want to give configure flask limiter to use a different db name than the generic "limits". But even if I change the URI its not changing. IDK what I am doing wrong any help is appreciated. 
Note: I am using mongodb.
```py
def get_ip_from_x_header():
    x_ip = flask.request.headers.get("X-IP")
    if x_ip:
        return x_ip
    return get_remote_address()


limiter = Limiter(
    app,
    key_func=get_ip_from_x_header,
    default_limits=["1000 per minute", "200 per second"],
    storage_uri=config.DBLINK + "Galam_Limiter",
    strategy="fixed-window",  # or "moving-window"
)

```

**Comment by eci-jgaffney** [comment]
I have run into this requirement too. Our application will be using a backend (mongodb) that is shared by many applications. I would like to specify a database name that would give the database administrator a clue to which application the database is for if they ever run into a problem with. The "key_prefix" configuration option is a decent workaround for the time being, but a custom database name is preferred.
